samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,99 @@ nRF5340: Network core bootloader
 
-The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
-This sample is included automatically if the application in the nRF5340 network core has the :option:`CONFIG_SECURE_BOOT` option set.
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+You can include the sample in your multi-image build in two different ways:
 
-A new set of firmware update files is generated by the build system when this image is included in a multi-image build.
-These files are identical to those described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+* Enabling the :option:`CONFIG_SECURE_BOOT` option in the application that runs on the nRF5340 network core.
+  With this option enabled, the build system includes the sample automatically.
+* Enabling the :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53` options in the application that runs on the application core.
+  In a multi-image build where there is also an application running on the network core, enabling MCUboot in the application core automatically includes the network core bootloader.
+  As such, with these options enabled, the build system automatically enables the :option:`CONFIG_SECURE_BOOT` option for the network core application.
 
-Testing
-=======
+  In addition, you must also enable the :ref:`subsys_pcd` library for the MCUboot image, enabling the option :option:`CONFIG_PCD`.
 
-To test the network core bootloader sample run the following commands:
+  .. note::
 
-#. |connect_terminal|
+     The firmware is signed exactly like the application firmware is signed.
+     As such, the firmware is also verified by MCUboot on the application core.
+     See :ref:`subsys_pcd` for more details.
 
-   Note that on the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+When you include this image in a multi-image build, the build system generates a new set of firmware update files
+These files match the ones described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
 
-#. ``west build -b nrf5340dk_nrf5340_cpuapp -d build_netboot samples/bluetooth/peripheral_uart -f -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_PCD=y``
+.. |sample path| replace:: :file:`samples/nrf5340/netboot/`
+
+.. include:: /includes/build_and_run.txt
+
+See also the following pages:
+
+* :ref:`ug_nrf5340_building` for additional details on building samples on the nRF5340 SoC.
+* :ref:`configure_application` for additional details on configuring your application.
```

Comment:
This feels a bit like throwing random information at the user and telling them to figure out how to do it. ;)
You don't need to include the general instructions for building and running if we have specific ones for nRF5340. And we haven't told users anything about configuring yet, so these are not "additional details".

So instead something like:
"See :ref:`configure_application` for information on how to enable the required configuration option.
Then follow the instructions in :ref:`ug_nrf5340_building` to build and program the images for the network and application core."

Reply:
Right, I was unsure if I could get rid of the include. Fixed